support-e2e/package.json

```diff
@@ -24,10 +24,12 @@
 	},
 	"devDependencies": {
 		"@guardian/prettier": "^8.0.0",
+		"@types/node": "^22.10.1",
 		"dotenv": "^16.3.1",
 		"husky": "^9.0.11",
 		"lint-staged": "^15.2.2",
-		"prettier": "^3.0.3"
+		"prettier": "^3.0.3",
+		"typescript": "^5.7.2"
```

Comment:
This has been added as there's a change in this PR to allow running `yarn tsc` in the support-e2e project to check the types. Playwright itself [doesn't check types](https://playwright.dev/docs/test-typescript) and I found myself wanting to do this. This could be pulled out to a separate PR if we preferred.

Reply:
If stays maybe a note in main conversation window on how to use?

Reply:
I've added a note to the README about being able to run `yarn tsc` to type check the tests.